src/AppProviders.tsx

```diff
@@ -69,13 +69,11 @@ export function AppProviders({ children }: ProvidersProps) {
                           <KeepKeyProvider>
                             <WalletConnectV2Provider>
                               <ModalProvider>
-                                <ErrorBoundary FallbackComponent={ErrorPage} onError={handleError}>
-                                  <ActionCenterProvider>
-                                    <AppProvider>
-                                      <DefiManagerProvider>{children}</DefiManagerProvider>
-                                    </AppProvider>
-                                  </ActionCenterProvider>
-                                </ErrorBoundary>
+                                <ActionCenterProvider>
```

Comment:
Error boundary currently broken with notification center bits (or at least with rfox claims), errors are swallowed and we're in the total darkness here - revert me once done, but in case you also lose a bit of sanity debugging things without errors thrown, this is the culprit. 

Not 100% sure whether that's only happening within notification center or any suspense things, but we should definitely dedicate some time to that, as currently, this makes devving virtually impossible

Reply:
Reverted, FYI @NeOMakinG ^^